packages/mobile/src/navigator/DrawerTopBar.tsx

```diff
@@ -29,14 +31,15 @@ function DrawerTopBar({ middleElement, rightElement, scrollPosition, testID }: P
     [scrollPosition]
   )
 
+  const hamburgerOnPress = () => {
+    ValoraAnalytics.track(HomeEvents.hamburger_tapped)
+    // @ts-ignore Only used in a drawer
+    return navigation.toggleDrawer()
+  }
+
   return (
     <Animated.View testID={testID} style={viewStyle}>
-      <TouchableOpacity
-        style={styles.hamburger}
-        // @ts-ignore Only used in a drawer
-        onPress={navigation.toggleDrawer}
-        hitSlop={iconHitslop}
-      >
+      <TouchableOpacity style={styles.hamburger} onPress={hamburgerOnPress} hitSlop={iconHitslop}>
```

Comment:
changing `onPress` from `navigation.toggleDrawer` to `hamburgerOnPress` (or even `() => { navigation.toggleDrawer() }`) causes the snapshot tests to fail with this diff:

```
          <View
            accessible={true}
    -       focusable={false}
    +       focusable={true}
            hitSlop={
              Object {
                "bottom": 10,
                "left": 10,
                "right": 10,
```

That's unexpected and can't figure out why it's happening.  @jeanregisser or @kathaypacific any ideas?